src/utils/file/index.js

```diff
@@ -24,10 +24,11 @@ export const convertFileToImage = src =>
   });
 
 // [1] - canvas를 이용하여 img를 Uint8ClampedArray로 변경
-export const convertImageToUint8ClampedArray = ($canvas, img) => {
-  $canvas.current.width = img.width;
-  $canvas.current.height = img.height;
-  const ctx = $canvas.current.getContext('2d');
+export const convertImageToUint8ClampedArray = (canvas, img) => {
```

Comment:
```suggestion
export const convertImageToUint8ClampedArray = (imgInfo, img) => {
```

굳이 아래 선언 없이 이렇게 하면 안되나요??

Reply:
no param reassign, 해당 린트 규칙으로 인해 수정하였습니다.
차후에 본인이 업로드한 사진을 사용자에게 노출할 필요가없으면 굳이 canvas를 parameter로 안받고, 함수 내부에서 만들어서 사용해도 될 것 같습니다.

Reply:
좋습니다
머지하십쇼